PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/PrebidMobile.java

```diff
@@ -201,12 +205,24 @@ public static HashMap<String, String> getCustomHeaders() {
 
 
     /**
-     * Initializes the main SDK classes. Makes request to Prebid server to check its status.
-     * You have to set host url ({@link PrebidMobile#setPrebidServerHost(Host)}) before calling this method.
+     * Initializes the main SDK classes and makes request to Prebid server to check its status.
+     * Status request is needed to understand whether server works and responds.
+     * You have to set host url ({@link PrebidMobile#setPrebidServerHost(Host)}) and
+     * /status endpoint url ({@link PrebidMobile#setCustomStatusEndpoint(String)}) before calling this method.
+     * <p>
+     * Calls SdkInitializationListener callback with enum initialization status parameter:
+     * <p>
+     * SUCCEEDED - initialization and status request were successful.
+     * <p>
+     * SERVER_STATUS_WARNING - initialization was successful but the status request was failed.
+     * <p>
+     * FAILED - initialization failed.
```

Comment:
Lets replace to 

```
FAILED - Prebid SDK is failed to initialize and is not able to work.
```

Also need to move this staus to the second position in the description, just after `SUCCEEDED`.